main.py

```diff
@@ -1,7 +1,67 @@
 import init_django_orm  # noqa: F401
 
-from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
-def main() -> QuerySet:
-    pass
+genres = [
+    "Genre",
+    "Action",
+    "Dramma"
+]
+
+actors = [
+    (
+        "George",
+        "Kianu",
+        "Scarlett",
```

Comment:
Organize data in `actors` variable in another way: 
```suggestion
actors = [
    ("George", "Klooney"),
    ("Kianu", "Reaves"),
    ("Scarlett", "Keegan"),
```
In this way, you could later unpack values correctly.